eth/main.cpp

```diff
@@ -829,6 +829,7 @@ int main(int argc, char** argv)
         }
     }
 
+    setupLogging(g_logVerbosity);
```

Comment:
Idea: We can create a common logging program options group and handle this with the notification.

Reply:
Program options group is a good idea, though I imagine we still would explicitly call `setupLogging` in each executable, it will have more parameters for other filtering (at least by channel)